doc/usecases/bindings/UC_new_application.md

```diff
@@ -0,0 +1,17 @@
+# Use Case: Programming Language Bindings
+
+## Summary
+
+- **Scope:** API
+- **Level:** Sea
+- **Actors:** API User
+- **Brief:** Developer wants to use libelektra in a new application written in another language than C
+- **Status:** Draft
```

Comment:
Why is the status "draft"?
Aren't there already some existing and working bindings which can fulfill the use case?

```suggestion
- **Status:** Implemented
```